site/class-1-play/bjorn2123.svg

```diff
@@ -0,0 +1,137 @@
+<svg xmlns="http://www.w3.org/2000/svg">
+  <!--Update the title with your username-->
+  <title>@Bjorn2123</title>
+
+    <circle
+            cx = "120"
+            cy = "120"
+            r = "100"
+            />
+
+    <rect
+          x = "250"
+          y = "20"
+          height="200"
+          width="400"
+          stroke="red"
+          stroke-width="20"
+    />
+    
+    <line 
+          x1 = "140"
+          y1="300"
+          x2="180"
+          y2="500"
+          stroke="aqua"
+          stroke-width="13"
+          />
+    
+   
+    
+   <g
+      stroke="antiquewhite"
+      stroke-width="20"
+      >
+       <polygon points="250,60 300,400 90,300"/>
+  <path d="M150 0 L75 200 L225 200 Z" />
+ </g> 
+    
+     <text x="350" y="150" fill="blue" >I love SVG!</text>
+<svg width="500" height="200" viewBox="0 0 50 40" >
+
+    <rect x="20" y="10" width="10" height="5"
+          style="stroke: #9c9c9c;"
+          font-family="inherit"
+          opacity="0.5"
+          fill="blue"
+          />
+    
+
+</svg>
+    
+    
+  <!-- NOTHING TO SEE HERE... MOVE ALONG!!! -->
```

Comment:
Could you remove this script, and its contents?

Also: You’ve got two opening `<svg>` tags, and three closing ones!